opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -409,8 +409,8 @@ def shutdown(self) -> None:
         self.done = True
         with self.condition:
             self.condition.notify_all()
-        self.worker_thread.join()
         self.span_exporter.shutdown()
+        self.worker_thread.join()
```

Comment:
I might be missing something here but isn't there reason we `join` before shutting down is because we want to wait for all the current telemetry in the queue to be exported before shutting down the exporter?